### PR TITLE
Update install.md

### PR DESCRIPTION
**Pull request by bezik**
Minor Russian-language related fixes (commas, dashes, etc.)

### Changelog category
- Documentation

### CODE REVIEW

File: docs/ru/getting-started/install.md [hunk 1]
```diff
@@ -230,20 +230,20 @@ sudo ./clickhouse install
...
+Для компиляции ClickHouse вручную используйте инструкцию для [Linux](../development/build.mdx) или [Mac OS X](../development/build-osx.md).
 
 Можно скомпилировать пакеты и установить их, либо использовать программы без установки пакетов. Также при ручой сборке можно отключить необходимость поддержки набора инструкций SSE 4.2 или собрать под процессоры архитектуры AArch64.
 
     Client: programs/clickhouse-client
     Server: programs/clickhouse-server
 
-Для работы собранного вручную сервера необходимо создать директории для данных и метаданных, а также сделать их `chown` для желаемого пользователя. Пути к этим директориям могут быть изменены в конфигурационном файле сервера (src/programs/server/config.xml), по умолчанию используются следующие:
+Для работы собранного вручную сервера необходимо создать каталоги для данных и метаданных, а также сделать сменить их владельца на желаемого пользователя ( `chown`). Пути к этим каталогам могут быть изменены в конфигурационном файле сервера (src/programs/server/config.xml), по умолчанию используются следующие:
```
Comment (alexey-milovidov) [inline docs/ru/getting-started/install.md:246]: Excessive whitespace.

File: docs/ru/getting-started/install.md [hunk 2]
```diff
@@ -258,7 +258,7 @@ sudo ./clickhouse install
 sudo service clickhouse-server start
 ```
 
-Смотрите логи в директории `/var/log/clickhouse-server/`.
+Смотрите журналы в директории `/var/log/clickhouse-server/`.
```
Comment (alexey-milovidov) [inline docs/ru/getting-started/install.md:261]: Use the term "логи".

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]
.

**Review by alexey-milovidov — APPROVED** [review]

**Comment by robot-ch-test-poll2** [comment]
*This is an automated comment for commit 9fae0e06d864dc0643b73c098892d72760a754a1 with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/56094/9fae0e06d864dc0643b73c098892d72760a754a1/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td><a href="https://github.com/ClickHouse/ClickHouse/actions/runs/6677554615">✅ success</a></td></tr>
<tbody>
</table>
</details>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/56094/9fae0e06d864dc0643b73c098892d72760a754a1/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

**Pull request merged**